documentation/src/main/markdown/packagemgrs/nuget.md

```diff
@@ -102,6 +102,8 @@ The NuGet inspectors derive dependency information from the first type of file i
 4. project.json
 5. XML of the project file
 
+In addition to the packages and the dependencues found from the above files, packages and dependencies will be included from a different project.assets.json if configured in the corresponding project's property file. (`<projectname>.<projectfiletype>.nuget.g.props`).
```

Comment:
Little typo in the word "dependencies".

Suggested text:
In addition to the packages and dependencies found from the above files, packages and dependencies will be included from a different project.assets.json if configured in the corresponding project's property file. (`<projectname>.<projectfiletype>.nuget.g.props`).